api_v3/lib/KalturaBaseUserService.php

```diff
@@ -293,6 +294,60 @@ protected function validateApiAccessControlByEmail($email)
 			$this->validateApiAccessControl($loginData->getLastLoginPartnerId());
 		}
 	}
+
+	/**
+	 * check if there were more than max_allowed calls for different resources
+	 * @param string $email
+	 * @throws KalturaErrors::FAILED_TO_INIT_OBJECT
+	 */
+	protected function validateRequestsAmount($email)
+	{
+		$cache = kCacheManager::getSingleLayerCache(kCacheManager::CACHE_TYPE_RESOURCE_RESERVATION);
```

Comment:
Add release note which states that this value must be set in cache.ini (for onprem users that did not set it so far)